### DISCUSSION TITLE
Autostart KE when specific external keyboard ist connected

### DISCUSSION THREAD

**Discussion by Dirk71** [pqrs-org/Karabiner-Elements#3772] [General]
Is there a way to autostart KE when a specific external keyboard is connected (e.g. Logitech MX Keys)....and quit KE when the external keyboard is disconnected?

**Comment by janstieler** [comment]
Normally this should not be necessary because the KE setings are mappt to a device and not to the sytsemwide.
Or what is your resulting problem?
